_targets/build.common.imxrt

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+#
+# Common shell script's functions for imxrt
+#
+# Copyright 2021 Phoenix Systems
+# Author: Hubert Buczynski
+#
+[ "${BASH_SOURCE[0]}" -ef "$0" ] && echo "You should source this script, not execute it!" && exit 1
+
+
+# Function gets the following argument:
+# - $1 - array contains user's commands
```

Comment:
`$1` is not an array but the first command.
It should be:
`# - $1 ... $n users's commands`

If you want pass array as argument then this can be done via reference for example:
```
b_mkscript_user() {
    declare -n cmds=$1
}
```
and
```
b_mkscript_user DEV_USER_SCRIPT
```

Reply:
Passing array via reference would be a better method, so I've changed to your proposition.
Thanks.